mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule PhoenixSlim.Mixfile do
   end
 
   def deps do
-    [{:phoenix, "~> 0.16"},
+    [{:phoenix, "~> 1.0.0"},
```

Comment:
Should we make this something like `{:phoenix, "> 0.16.0"}`?  Are there any disadvantages?  The upside is fewer releases.